lib/appliance_console/database_configuration.rb

```diff
@@ -83,7 +83,7 @@ def create_region
 
     def join_region
       require 'tempfile'
-      temp = Tempfile.new(RAILS_ROOT.join("tmp").to_s)
+      temp = Tempfile.new(%w(joinregion .rb), RAILS_ROOT.join("tmp").to_s)
```

Comment:
I find the %w makes this hard to read...it looks like you typoed it.  I think `["joinregion", ".rb"]` would be clearer.  But overall, it's really minor.